polkadot/xcm/xcm-builder/src/tests/mock.rs

```diff
@@ -772,3 +773,122 @@ pub fn fungible_multi_asset(location: Location, amount: u128) -> Asset {
 pub fn fake_message_hash<T>(message: &Xcm<T>) -> XcmHash {
 	message.using_encoded(sp_io::hashing::blake2_256)
 }
+
+/// A dummy `DenyExecution` impl which returns `ProcessMessageError::Yield` when XCM contains
+/// `ClearTransactStatus`
+pub struct DenyClearTransactStatusAsYield;
```

Comment:
@yrong I think the last one :), please move these impl directly to the `polkadot/xcm/xcm-builder/src/tests/barriers.rs`, and change `pub struct` -> `struct` (not sure if compiles)

Reply:
https://github.com/paritytech/polkadot-sdk/pull/7169/commits/8b949639e928519661f1b082ceb53d284239a6fe